solr/CHANGES.txt

```diff
@@ -164,7 +164,8 @@ Dependency Upgrades
 
 Other Changes
 ---------------------
-(No changes)
+* SOLR-17579: Remove unused code and other refactorings in ReplicationHandler and tests.  Removed unused public 
```

Comment:
honestly, would any reader of CHANGES.txt care?  I'm not questioning your effort in this PR but ...

Reply:
i went back and forth on it, since I'd created a JIRA...   I then thought, it should then go in Changes...    Plus removing a public variable...

We do need to add to our dev docs a concise "this is what Changes" is for...